src/js/components/stock-movement-wizard/AddItemsPage.jsx

```diff
@@ -389,8 +390,31 @@ class AddItemsPage extends Component {
    * @public
    */
   getLineItemsToBeSaved(lineItems) {
```

Comment:
I think this should be refactored a bit now because you are mixing to much the `lineItemsToBeAdded` with `lineItemsToBeUpdated` and it get very hard to read. At least comment your steps, please.

Reply:
@awalkowiak I have added comments